### ISSUE TITLE
v2.2.2-rc7 Lost Active Apps and online Apps after a day

### ISSUE THREAD

**Issue by pz1** [Z-Wave-Me/zwave-smarthome#125]
FWIW: After a day of usage of **v2.2.2-rc7** on my minor production machine, I noticed that the App tabs *Online* and *Active* were empty. 
Nothing notable in my log file. Also a reboot of this RaspberryPi B with 3gen RaZberry did not cure the problem.

After reinstall of 2.2.1 everything was there again.

Linux raspberrypi 4.1.13+ #826 PREEMPT Fri Nov 13 20:13:22 GMT 2015 armv6l GNU/Linux

**Comment by ronie-z-wave** [comment]
hmm sounds not cool ... but we didn't observed this yet, please try to give us more information if you make this experience again

**Comment by pz1** [comment]
Sent the log to niels' email account

**Comment by PoltoS** [comment]
Please send also browser console log. These problems are usually due to some exception in JS in the UI.

**Comment by pz1** [comment]
I am exchanging with Niels. A.o. I did sent the console log. The pattern below occured if I clicked under Apps on any of the three TABs
```
Error: $scope.dataHolder.modules.ids[item.modulename].version.split is not a function
setOnlineModules/onlineModules<@http://myIP:8083/smarthome/app/js/build.js:16262:52
m.select/<@http://myIP:8083/smarthome/app/js/build.js:3272:2851
m.forEach@http://myIP:8083/smarthome/app/js/build.js:3272:2350
m.select@http://myIP:8083/smarthome/app/js/build.js:3272:2826
m.mixin/</m.prototype[t]@http://myIP:8083/smarthome/app/js/build.js:3272:15543
setOnlineModules@http://myIP:8083/smarthome/app/js/build.js:16239:29
$scope.allSettled/<@http://myIP:8083/smarthome/app/js/build.js:16133:17
De/e/l.promise.then/J@http://myIP:8083/smarthome/app/js/build.js:7583:87
De/f/<.then/<@http://myIP:8083/smarthome/app/js/build.js:7584:259
Yd/this.$get</h.prototype.$eval@http://myIP:8083/smarthome/app/js/build.js:7595:28
Yd/this.$get</h.prototype.$digest@http://myIP:8083/smarthome/app/js/build.js:7592:109
Yd/this.$get</h.prototype.$apply@http://myIP:8083/smarthome/app/js/build.js:7595:360
m@http://myIP:8083/smarthome/app/js/build.js:7554:452
w@http://myIP:8083/smarthome/app/js/build.js:7559:463
ye/</B.onreadystatechange@http://myIP:8083/smarthome/app/js/build.js:7561:24
 build.js:7574:67 

e/<() build.js:7574
Qd/this.$get</<() build.js:7550
De/e/l.promise.then/J() build.js:7583
De/f/<.then/<() build.js:7584
Yd/this.$get</h.prototype.$eval() build.js:7595
Yd/this.$get</h.prototype.$digest() build.js:7592
Yd/this.$get</h.prototype.$apply() build.js:7595
m() build.js:7554
w() build.js:7559
ye/</B.onreadystatechange() 
```

**Comment by ronie-z-wave** [comment]
Thats an ui, problem we'll fix this ... do you have modules installed that has no version in modules.json?

**Comment by pz1** [comment]
Found indeed Maros Kollar's RemoteHost App in UserModules section, which I downloaded from the app store, but never used.
It has at the end of the file":
   "version" : 1.02
unquoted

After removal of that module from userModules and Zway restart, the bug has gone. Thanks.
As far as I am concerned this issue may be closed.

**Comment by ronie-z-wave** [comment]
fixed with now released v2.2.2 (11.04.16)
